Remotion/Web/Development.WebTesting.IntegrationTests/BrowserSessionTest.cs

```diff
@@ -75,12 +75,13 @@ public void GeckoDriver_DoesNotSupportBrowserLogs ()
 
       var home = Start();
 
-      var selenium = (IWebDriver)home.Context.Browser.Driver.Native;
+      var js = JavaScriptExecutor.GetJavaScriptExecutor(home.Context.Browser);
+      js.ExecuteScript($"console.error('Error')");
+      js.ExecuteScript($"console.warn('Warning')");
+
+      var logs = ((IWebDriver)home.Context.Browser.Driver.Native).Manage().Logs.GetLog(LogType.Browser);
 
-      Assert.That(
-          () => selenium.Manage().Logs.GetLog(LogType.Browser),
-          Throws.InstanceOf<NullReferenceException>().With.Message
-              .EqualTo("Object reference not set to an instance of an object."));
+      Assert.That(logs, Is.Empty);
```

Comment:
I read something in the release notes about selenium fixing logging issues. The only fixed the exception when getting the logs, but firefox still doesn't return logs? Could you please check the release notes what they say about logging?

Reply:
FirefoxDriver doesn't implement ISupportLogs like other BrowserDrivers, so there is no support for reading logs at this point. The release notes only mention retrieving logs in relation to Chrome.

Reply:
okay, so the only improvement was to get rid of the exception. okay.